api/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "contains all the APIs used by the SCE website",
   "main": "SceHttpServer.js",
   "scripts": {
-    "start": "node SceHttpServer.js",
+    "start": "nodemon util/SceHttpServer.js",
```

Comment:
Are we still planning on creating one package.json file for all the different subdivisions?

Reply:
We can, it is creates some really annoying errors. For now we can keep it like this and when #356 is finished by @SOMLpj we can give it a shot